modules/ephemeral-storage-shared-resource-csi-driver-operator.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: CONCEPT
+
+[id="ephemeral-storage-shared-resource-csi-driver-operator_{context}"]
+= Shared Resource CSI Driver operator
+
+[role="_abstract"]
+As a cluster administrator, you can use the Shared Resource CSI Driver in {product-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, builds can use the contents of those `Secret` or `ConfigMap` objects across potentially any namespace in the cluster.
```

Comment:
```suggestion
As a cluster administrator, you can use the Shared Resource CSI Driver in {product-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, Pods, the other Kubernetes types that expose Volume mounts, as well as {product-title} Builds can securely use the contents of those `Secret` or `ConfigMap` objects across potentially any namespace in the cluster.  Hence, there are currently two types of "Shared Resources".  A `SharedSecret` is using for `Secrets`, and a `SharedConfigMap` is used for `ConfigMaps`.
```

Reply:
Thanks. Updated. In keeping with the [API object formatting
](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) section in our "Documentation guidelines," I made some minor adjustments:
```
To accomplish this, there are currently two types of shared resources: a `SharedSecret` resource for `Secret` objects, and a `SharedConfigMap` resource for `ConfigMap` objects.
```
Basically, Red Hat documentation doesn't make an object between backticks plural by adding an s to it. We add a label after the object and make that label plural. For example, we write "`Secret` objects," not "`Secrets`."